test_runner/fixtures/neon_fixtures.py

```diff
@@ -1441,6 +1441,7 @@ def neon_env_builder(
     pageserver_virtual_file_io_engine: str,
     pageserver_default_tenant_config_compaction_algorithm: Optional[Dict[str, Any]],
     pageserver_aux_file_policy: Optional[AuxFileStore],
+    record_property: Callable[[str, object], None],
```

Comment:
Don't we need to define this as a fixture somewhere?

Reply:
It's a built-in pytest fixture (we already use it in benchmarks): https://docs.pytest.org/en/7.1.x/reference/reference.html#record-property

Reply:
Thanks. I wasn't expecting that :)